layout-repr/build.rs

```diff
@@ -0,0 +1,64 @@
+//! Build script to generate our glyph name lookup table.
```

Comment:
Given our use case what if we give you a glyph order and we delete the aglfn.txt bit entirely?

Reply:
> Other parts of fontc that generate tables like this use python, e.g. `resources/scripts/non_spacing_marks.py`. It would be nice if this was consistent.

That's fair, but I'd prefer not to take this working build script (which is, in any case, the canonical way to do this sort of thing in a rust project) and port it to python right now. I'm happy to open an issue to look into standardizing this sort of thing, though.


> Given our use case what if we give you a glyph order and we delete the aglfn.txt bit entirely?

I presume you mean a glyph order that we take from the source? My thinking for this tool was that it should work entirely on a font binary, without assuming access to a source, but this isn't 100% necessary.

I _suspect_ that on a long enough time frame we are going to want a way to get sensible names for glyphs in a binary font, and I _suspect_ that this will end up being a standalone crate that lives in `fontations` or elsewhere, but I don't have strong feelings here.

Given that this code exists and works I'd lean towards just leaving it for now? Are there any advantages to removing it that I might be overlooking?